### DISCUSSION TITLE
Add semantic label to the segmentation visualization results 分割可视化结果中加上语义信息

### DISCUSSION THREAD

**Discussion by vansinhu** [open-mmlab/OpenMMLabCamp#154] [MMSegmentation]
# Add semantic label to the segmentation visualization results 
 分割可视化结果中加上语义信息 

The current segmentation result visualization content is just assigning a specified color to each semantic segmentation area, there is no category information, the task is to add the segmentation category information to the visualization result, details can be found at [[https://github.com/open-mmlab/mmsegmentation/issues/2578](https://github.com/open-mmlab/mmsegmentation/issues/2578)](https://github.com/open-mmlab/mmsegmentation/issues/2578](https://github.com/open-mmlab/mmsegmentation/issues/2578)) 

当前分割结果可视化内容是只是给每一个语义分割区域分配指定颜色，没有类别信息，该任务是希望将分割类别信息添加到可视化结果中。
 
 详细可见[https://github.com/open-mmlab/mmsegmentation/issues/2578](https://github.com/open-mmlab/mmsegmentation/issues/2578)

## Task Information

**Repository**：MMSegmentation

**Difficulty**：Medium

**Points**：30

**Estimated Time Consumption**：14 days

## Task Status

**Claimant's Github ID**：CastleDream

**Claim Time**：2023年07月17日

**PR Link**：https://github.com/open-mmlab/mmsegmentation/pull/3229

## Points to Note

❗️请注意：此任务属于超级视客营任务，不能被领取！请登陆 [超级视客营平台](https://openmmlab.com/activity/codecamp) 按照流程申请任务，我们会尽快审核～

**Comment by CastleDream** [comment]
Claim the mission

**Comment by CastleDream** [comment]
[[CodeCamp2023-154] Add semantic label to the segmentation visualization results #3229](https://github.com/open-mmlab/mmsegmentation/pull/3229)

**Reply by crazysteeaam** [reply]
good job
